src/main/java/net/mcreator/element/converter/ConverterRegistry.java

```diff
@@ -66,7 +67,8 @@ public class ConverterRegistry {
 				new BiomeFrozenTopLayerConverter()));
 		put(ModElementType.OVERLAY, Collections.singletonList(new OverlayCoordinateConverter()));
 		put(ModElementType.BLOCK,
-				Arrays.asList(new BlockLuminanceFixer(), new BlockBoundingBoxFixer(), new BlockLightOpacityFixer()));
+				Arrays.asList(new BlockLuminanceFixer(), new BlockBoundingBoxFixer(), new BlockLightOpacityFixer(),
+						new BlockRequiresCorrectToolConverter()));

```

Comment:
I'd name the new one without using verbs like it's done to all the other converters so far, maybe `BlockToolRequirementConverter`.

Reply:
I think this name is fine, since `requiresCorrectTool` is the name of the field